src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/RuntimeEventSource.cs

```diff
@@ -50,6 +55,17 @@ public static void Initialize()
         // as you can't make a constructor partial.
         private RuntimeEventSource(int _) { }
 
+        private enum EventId : int
+        {
+            AppContextSwitch = 1
+        }
+
+        [Event((int)EventId.AppContextSwitch, Level = EventLevel.Informational, Keywords = Keywords.AppContext)]
+        internal void LogAppContextSwitch(string switchName, int value)
```

Comment:
Is having the `Log` prefix in the event name normal? I think I have more commonly seen names like just `AppContextSwitch` or `AppContextSwitchInfo`.

Reply:
It's internal so I didn't think too hard. We can certainly change it if it becomes annoying later.

Reply:
I was thinking it was external in that it is the name of the event that gets fired.